test/plugins/webhooks/github.test.js

```diff
@@ -253,6 +253,16 @@ describe('github plugin test', () => {
                     assert.equal(reply.statusCode, 500);
                 });
             });
+
+            it('handles checkouting when given a non-listed user', () => {
+                userFactoryMock.get.resolves(null);
+                userFactoryMock.get.withArgs({ username: 'sd-buildbot' }).resolves(userMock);
+
+                return server.inject(options)
+                .then((response) => {
+                    assert.equal(response.statusCode, 201);
```

Comment:
I agree, but it's always better to assert. If somehow the code was okay with `null` user. It's better to have this check. :)